work-tracker-spring/src/main/java/com/deere/isg/worktracker/spring/SpringWork.java

```diff
@@ -113,7 +114,7 @@ protected void setHttpMethod(String httpMethod) {
     }
 
     protected String buildValueRegex(String value) {
-        return "/" + WHITE_SPACE_REGEX + value + WHITE_SPACE_REGEX + "/";
+        return "/" + WHITE_SPACE_REGEX + quote(value) + WHITE_SPACE_REGEX + "/";
```

Comment:
You can also do:
`return quote("/" + WHITE_SPACE_REGEX .....);`

Reply:
Actually, you can't because all the whitespace regex would not find regex and generally would break everything.